src/chmod/chmod.rs

```diff
@@ -229,7 +230,7 @@ impl Chmoder {
     fn change_file(&self, fperm: u32, mode: u32, file: &Path, path: &str) -> Result<(), i32> {
```

Comment:
I think you also need to mark this as `#[cfg(unix)]` because you use `display_permissions_unix()`.